<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>RSS links:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rss_link">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>591</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>title:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rss_link_title">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>451</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="save_new_link">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>120</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>saved rss souces:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="recent_rss_links">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 18px;
text-align: center;</string>
   </property>
   <item>
    <property name="text">
     <string>-- There's no rss links --</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="parse_link">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>230</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Parse</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
